lib/employees/resolvers.ts

```diff
@@ -119,3 +123,41 @@ export const updateEmployee: Resolver = async (_, args, { prisma }) => {
     ok: true,
   };
 };
+
+/**
+ * Delete an employee
+ * @returns status of operation (ok)
+ */
+export const deleteEmployee: Resolver = async (_, args, { prisma }) => {
+  const id = parseInt(args.id);
+  const employee = await prisma.employee.findUnique({
+    where: {
+      id: id,
+    },
+    select: {
+      active: true,
+    },
+  });
+
+  if (employee == null) {
```

Comment:
```suggestion
  if (employee === null) {
```
This will give us type comparison as well.